app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -433,8 +433,11 @@ def rbac_group_seq_edit_screen
     end
     @edit[:key] = "rbac_group_edit__seq"
     @edit[:current] = copy_hash(@edit[:new])
+
     @right_cell_text = _("Editing Sequence of User Groups")
-    @tabs = [["ldap_seq_edit", "Edit Sequence of User Groups"], ["ldap_seq_edit", "Edit Sequence of User Groups"]]
+    @active_tab = "ldap_seq_edit"
```

Comment:
@hayesr it is safe to remove setting of @tabs here as they aren't being used. We no longer show tabs on screens that only have single tab.